lib/Backend/IR.cpp

```diff
@@ -2859,7 +2860,7 @@ Instr::FindRegUse(StackSym *sym)
         else if (src1->IsIndirOpnd())
         {
             IR::IndirOpnd *indirOpnd = src1->AsIndirOpnd();
-            if (indirOpnd->GetBaseOpnd()->m_sym == sym)
+            if (indirOpnd->GetBaseOpnd() && indirOpnd->GetBaseOpnd()->m_sym == sym)
```

Comment:
Could be null in case addition is with constant. There we will need index * scale + displacement.